tests/Propel/Tests/Runtime/Map/TableMapTest.php

```diff
@@ -58,7 +58,7 @@ public function testConstructor()
         try {
             $tmap = new TableMap();
             $this->assertTrue(true, 'A table map can be instantiated with no parameters');
-        } catch (Exception $e) {
+        } catch (\Exception $e) {
```

Comment:
Can we revert this?
It should pick it up from the use statement above, no?

Reply:
Removed the backslash and added the missing `use`. Better?